Adyen/ApiSerialization/SaleToPoiMessageSecuredSerializer.cs

```diff
@@ -7,14 +7,14 @@
 
 namespace Adyen.ApiSerialization
 {
-    internal class SaleToPoiMessageSecuredSerializer
+    public class SaleToPoiMessageSecuredSerializer
     {
-        internal string Serialize(SaleToPoiMessageSecured saleToPoiMessageSecured)
+        public string Serialize(SaleToPoiMessageSecured saleToPoiMessageSecured)
```

Comment:
Good one. I believe there was a merchant requesting this to be public, and I don't see a reason why not too so good move here.